homeassistant/components/webmin/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""The Webmin integration."""
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import WebminUpdateCoordinator
+
+PLATFORMS = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Webmin from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = WebminUpdateCoordinator(
+        hass, entry
+    )
```

Comment:
Please assign the coordinator to a local variable first, then do the first refresh, then assign it to hass.data. this way we dont have broken objects stored